nupic/frameworks/opf/metrics.py

```diff
@@ -709,26 +734,14 @@ def __init__(self, metricSpec):
     # Construct moving average instance
     self._movingAverage = MovingAverage(self.mean_window)
 
-  def getMetric(self):
-    return self._subErrorMetrics[0].getMetric()
-
   def addInstance(self, groundTruth, prediction, record = None):
-
-    # If missing data,
-    if groundTruth == SENTINEL_VALUE_FOR_MISSING_DATA:
-      return self._subErrorMetrics[0].aggregateError
-
-    if self.verbosity > 0:
-      print "groundTruth:\n%s\nPredictions:\n%s\n%s\n" % (groundTruth, prediction, self.getMetric())
-
     # Use ground truth from 'steps' steps ago as our most recent ground truth
     lastGT = self._getShiftedGroundTruth(groundTruth)
     if lastGT is None:
       return self._subErrorMetrics[0].aggregateError
-
     mean = self._movingAverage(lastGT)
-
-    return self._subErrorMetrics[0].addInstance(groundTruth, mean, record)
+    return super(MetricMovingMean, self).addInstance(groundTruth, mean, record)
+###############################################################################
```

Comment:
If you think this smugled line is really necessary, then just keep it in the midle betwen the 2 methods:

```
def function1():
  return ....


########################


def function2:
  ...
````